features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/MessagesReactionButton.kt

```diff
@@ -39,9 +40,9 @@ import io.element.android.libraries.designsystem.theme.components.Surface
 import io.element.android.libraries.designsystem.theme.components.Text
 
 @Composable
-fun MessagesReactionButton(reaction: AggregatedReaction, modifier: Modifier = Modifier) {
+fun MessagesReactionButton(reaction: AggregatedReaction, modifier: Modifier = Modifier, onClick: () -> Unit) {
     Surface(
-        modifier = modifier,
+        modifier = modifier.clickable { onClick() } ,
```

Comment:
nit: you should be able to just use a method reference instead of a lambda like `modifier.clickable(onClick)`